study-builder/studies/atcp/substitutions.conf

```diff
@@ -0,0 +1,247 @@
+{
+  "i18n": {
+    "de": {
+      include required("i18n/de.json")
+    },
+    "en": {
+      include required("i18n/en.json")
+    },
+    "es": {
+      include required("i18n/es.json")
+    },
+    "fr": {
+      include required("i18n/fr.json")
+    },
+    "hi": {
+      include required("i18n/hi.json")
+    },
+    "it": {
+      include required("i18n/it.json")
+    },
+    "ja": {
+      include required("i18n/ja.json")
+    },
+    "pl": {
+      include required("i18n/pl.json")
+    },
+    "pt": {
+      include required("i18n/pt.json")
+    },
+    "ru": {
+      include required("i18n/ru.json")
+    },
+    "tr": {
+      include required("i18n/tr.json")
+    },
+    "zh": {
+      include required("i18n/zh.json")
+    }
+  },
+  "subs": {
+    "_includes": {
+      "answer_templates": {
+        "feeding" : { include required("./snippets/feeding/feeding-answers.conf")},
+        "feeding_nested1" : { include required("./snippets/feeding/feeding-nested-answers1.conf")}
+      },
+      "required_block": { include required("./snippets/feeding/required-block.conf")},
+      "registation_countries": {
+        "countries_list": { include required("snippets/picklist-question-country-required-eligible-first.conf")}
+        "country_af": { include required("snippets/countries-with-states/picklist-question-Afghanistan-state-province-required.conf")}
+        "country_al": { include required("snippets/countries-with-states/picklist-question-Albania-state-province-required.conf")}
+        "country_dz": { include required("snippets/countries-with-states/picklist-question-Algeria-state-province-required.conf")}
+        "country_ad": { include required("snippets/countries-with-states/picklist-question-Andorra-state-province-required.conf")}
+        "country_ao": { include required("snippets/countries-with-states/picklist-question-Angola-state-province-required.conf")}
+        "country_ag": { include required("snippets/countries-with-states/picklist-question-Antigua and Barbuda-state-province-required.conf")}
```

Comment:
Why does some of these files have spaces in their filenames? It might be better to get rid of the spaces and replace with dash.

Reply:
Filenames were fixed